bucket/vroid-studio.json

```diff
@@ -0,0 +1,32 @@
+{
+    "version": "0.11.3",
+    "description": "A 3D character maker that lets you use your imagination to easily create your own original characters.",
+    "homepage": "https://vroid.com/en/studio",
+    "license": {
+        "identifier": "Proprietary",
+        "url": "https://policies.pixiv.net/en.html#vroidstudio"
+    },
+    "architecture": {
+        "64bit": {
+            "url": "https://download.vroid.com/dist/VRoidStudio-v0.11.3-win.zip",
+            "hash": "ff41ff00954ab7159412d8369b83dfbba617f96fb2b84a783b9254e1a4213cae",
+            "bin": "VRoidStudio.exe",
+            "shortcuts": [
+                [
+                    "VRoidStudio.exe",
+                    "VRoid Studio"
+                ]
+            ]
```

Comment:
```suggestion
            "hash": "ff41ff00954ab7159412d8369b83dfbba617f96fb2b84a783b9254e1a4213cae"
```

Reply:
I only removed the "bin" attribute. I left the shortcut there to have it searchable via the start menu of Windows.